gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/openapi-apis.yaml

```diff
@@ -5755,6 +5816,15 @@ components:
                 type: array
                 items:
                     type: string
+        eventTypes:
+            name: types
+            description: List of event types to filter on.
+            in: query
+            explode: false
+            schema:
+                type: array
+                items:
+                    type: string
```

Comment:
Not a blocker as the string is converted into an enum in the buildInput method but should we have a list of enum ?